lighthouse-core/runner.js

```diff
@@ -119,12 +119,14 @@ class Runner {
         reportCategories = ReportScoring.scoreAllCategories(opts.config.categories, resultsById);
       }
 
+      const finalUrl = artifacts.URL ? artifacts.URL.finalUrl : '';
```

Comment:
when do we not have one? isn't it always set now?

Reply:
lol i'm reading the opposite. since this PR nukes the URL gatherer how is `artifacts.URL` set?

Reply:
good catch. Left over from getting un-updated tests to run :)

only future issue for those of us using saved artifacts (aka @paulirish) is that this is a breaking saved artifacts change, and since it'll be "cannot read property 'finalUrl' of undefined", this will throw on any `-A` run with previously saved artifacts instead of producing an LHR with an empty `finalUrl`

Reply:
yup! no worries. :)